data/apis/api3/beacons/amberdata ust_usd.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "AmberData BTC/USD",
-  "description": "AmberData - BTC/USD - Pair",
-  "beaconId": "0x5bc49582dcc3c0ad2e4ca760909438e637484cc0f373e354bb19b23630056ce4",
+  "name": "AmberData UST/USD",
+  "description": "AmberData - UST/USD - Pair",
+  "beaconId": "0x472ccf005a6ee77277c6262321c4228aa31eabfb199ccd216ebc486302df0f4d",
   "airnodeAddress": "0x5b1243308f56156553F067D354b162f71E581b28",
-  "templateId": "0x648bac0274b2391e6bce95f712a66df856e8a4bf2a27f7fbfa17932218b10d92",
+  "templateId": "0x1bab8014c068e6871bf4043a74c51164d05d482d8cb6f1f66f8edcd7f3b6bb0f",
```

Comment:
So when trying to run export-ops-json-payloads script locally I get an error because it is not able to find a template with this id in data/apis/amberdata/templates. My main concern is that this doesn't seem to happen in CI 🤔

Reply:
That is curious?! I'll investigate.

Reply:
So we don't currently do cross-field validation when we validate (like does a templateId referenced in a beacon exist?). ZoD can do this (I've seen bits of it in Airnode), so this is something we need in operations pretty soon. This is referenced in [BEC-457](https://api3dao.atlassian.net/jira/software/projects/BEC/boards/31?selectedIssue=BEC-457)

In this case the template is actually missing, but it appears that the export-ops-json-payload.ts script doesn't exit with an error code when it encounters an error and thus CI is happy. I'll add a proper error code for exceptions in a try/catch and delete the offending beacon (UST/USD).

Once again, great find, thank you again.

Reply:
I've pushed up the change with the faulty file, it should fail in CI now.

Reply:
The build fails now if the export throws: https://github.com/api3dao/operations/runs/6622984206?check_suite_focus=true